src/sql/workbench/browser/modal/optionsDialogHelper.ts

```diff
@@ -91,7 +91,7 @@ export function getOptionValueAndCategoryValues(option: azdata.ServiceOption, op
 		if (option.valueType === ServiceOptionType.boolean) {
 			possibleInputs.push(trueInputValue, falseInputValue);
 		} else {
-			option.categoryValues.map(c => possibleInputs.push(c.name));
+			option.categoryValues.map(c => possibleInputs.push(c.displayName ?? c.name));
```

Comment:
You've verified that this correctly sends the name (value) to the tools service and not the display name?

Reply:
I initially tried this as a quick fix and I believe that @Charles-Gagnon is right. My result was that the display name was getting sent to the back end. Unfortunately that will not work at present.

Reply:
I see.

That's hilarious, i'll look into it more.